stdlib/file/src/main/ballerina/src/file/file.bal

```diff
@@ -78,10 +90,13 @@ function externRemove(handle path, boolean recursive) returns Error? = @java:Met
 
 # Renames(Moves) old path to new path.
 # If new path already exists and it is not a directory, this replaces the file.
+# ```ballerina
+# file:error? results = file:rename("/A/B/C", "/A/B/D");
+# ```
 #
-# + oldPath - String value of old file path.
-# + newPath - String value of new file path.
-# + return - Returns an `Error` if failed to rename.
+# + oldPath - String value of old file path
+# + newPath - String value of new file path
+# + return - An `Error` if failed to rename
```

Comment:
```suggestion
# + return - An `Error` if failed to rename
```
Do we need to specify the error?

Reply:
The encompassing Error object is named as `Error`, I've added the module name to be more clear. Added in dba6e94